openstudiocore/src/openstudio_lib/library/OpenStudioPolicy.xml

```diff
@@ -876,6 +876,12 @@
     <rule IddField="Outlet Node Name" Access="HIDDEN"/>
     <rule IddField="Heating Coil Name" Access="HIDDEN"/>
   </POLICY>
+  <POLICY IddObjectType="OS_ZoneHVAC_EquipmentList">
```

Comment:
Right now you just see it’s name. But once the PR I linked to gets merged in you’ll also see « Load Distribution »

Reply:
#3262

Reply:
Here's with 3262 merged in: 

![image](https://user-images.githubusercontent.com/5479063/45618866-89c40680-ba77-11e8-9543-b9da81d0da6c.png)

Because that field isn't explicitly set in CTor for ZoneHVACEquipmentList, it shows up as blue.